website/views/static/community-guidelines.jade

```diff
@@ -30,14 +30,14 @@ block content
         |&nbsp;
         =env.t('February')
         |&nbsp;9&comma; 2015
```

Comment:
The date should also be updated. If more than a day goes by before this is merged in, after you've adjusted it, I'll manually adjust it myself after it gets merged.